### PR TITLE
docs/trusted-publishers: example fixes, improvements

### PR DESCRIPTION
**Pull request by woodruffw**
This is a best practice, and it's what the suggested GitHub Action does.

CC @webknjaz

### CODE REVIEW

**Comment by woodruffw** [comment]
I've also rolled in @webknjaz's other suggestion re: step outputs instead of `GITHUB_ENV`.

**Review by webknjaz — APPROVED** [review]

File: docs/user/trusted-publishers/troubleshooting.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ title: Troubleshooting
...
+
+PyPI currently imposes somewhat conservative restrictions on trusted
+publisher registration: no more than 20 publishers can be registered by
+a single user or IP address within a 24 hour window.
+
+This should be more than sufficient for most users (since publisher
+registration should happen rarely relative to publisher use), but maintainers
+with large numbers of projects or who access PyPI via a shared IP address
+may run into ratelimiting errors. If this happens to you, please wait 24 hours,
+try again, and then contact PyPI's admins if the problem persists.
```
Comment (woodruffw) [inline docs/user/trusted-publishers/troubleshooting.md:19]: Flagging for review: this documents the current ratelimits, which are probably excessively conservative. I don't think there's any serious risk in bumping them from 20/day to 100/day, if we want to reduce the amount of maintainer interaction over this 🙂

**Comment by woodruffw** [comment]
NB: https://github.com/pypi/warehouse/pull/13367 should probably be merged first, since it contains a ratelimit change that this documents.

File: docs/user/trusted-publishers/troubleshooting.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ title: Troubleshooting
...
+
+PyPI currently imposes ratelimits on trusted publisher registration: no more
+than 100 publishers can be registered by a single user or IP address within a 24
+hour window.
+
+This should be more than sufficient for most users (since publisher
+registration should happen rarely relative to publisher use), but maintainers
+with large numbers of projects or who access PyPI via a shared IP address
+may run into ratelimiting errors. If this happens to you, please wait 24 hours,
+try again, and then contact PyPI's admins if the problem persists.
```
Comment (di) [inline docs/user/trusted-publishers/troubleshooting.md:19]: This should link to somewhere that describes how to contact admins.

**Review by di — APPROVED** [review]
LGTM aside from that one comment.

File: docs/user/trusted-publishers/troubleshooting.md (hunk 2 above)
Comment (woodruffw) [reply docs/user/trusted-publishers/troubleshooting.md]: The FAQs don't contain a dedicated "contact" section but they do link directly to the admin email, so I've done that here as well. Let me know if that works for you!

**Pull request merged**